UnityProject/Assets/Scripts/Items/Medical/Implanter.cs

```diff
@@ -135,14 +131,14 @@ public void ServerPerformInteraction(HandActivate interaction)
 			}
 			else
 			{
-				TogglePrimed();
+				TogglePrimed(true);
 			}
 		}
 	}
 
-	private void TogglePrimed()
+	private void TogglePrimed(bool isPrimed)
```

Comment:
If we're going to tell the method what boolean to use, consider renaming this to something like `SetPrimed()`.